data/json/items/magazine/38super.json

```diff
@@ -21,7 +21,7 @@
     "looks_like": "glock17_17",
     "type": "MAGAZINE",
     "name": { "str": "M1911 .38 Super 9-round magazine" },
-    "description": "A compact single stack box magazine for the M1911.",
+    "description": "A compact, single stack box magazine for an M1911 chambered for .38 Super rounds.",
```

Comment:
"chambered in .38 Super." works better i think

Reply:
Done